test/bigint-buffer.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const metatests = require('metatests');
+const {
+  writeBigIntToBuffer,
+  readBigIntFromBuffer
+} = require('../lib/bigint-buffer');
+
+const BUFFER_SIZE = 8;
+
+const bigints = [0xffffffn, 0xffffffffffn, 0xffffffffffffffffn];
+const buffers = [
+  Buffer.from([0xff, 0xff, 0xff, 0, 0, 0, 0, 0]),
+  Buffer.from([0xff, 0xff, 0xff, 0xff, 0xff, 0, 0, 0]),
+  Buffer.from([0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff]),
+];
+
+metatests.test('write bigint to buffer', test => {
+  let buffer = Buffer.alloc(BUFFER_SIZE);
+  writeBigIntToBuffer(bigints[0], buffer);
+
+  test.same(
```

Comment:
You should use `test.strictSame` instead of `test.same`. This applies to all places.